src/canmatrix/xls.py

```diff
@@ -309,6 +303,28 @@ def dump(db, file, **options):
     # save file
     workbook.save(file)
 
+############################ load ###############################
+
+def parse_value_name_collumn(value_name, value, signal_size, float_factory):
+    mini = maxi = offset = None
+    value_table = dict()
+    if ".." in value_name:
+        (mini, maxi) = value_name.strip().split("..", 2)
```

Comment:
```suggestion
        (mini, maxi) = value_name.strip().split("..")
```

The second argument is the maximum number of splits, not the maximum number of elements.  So `2` would allow for three elements which would fail here.  Since we want to fail on too many `..`s anyways, we can just do it here with the assignment.  Though a more explicit error could be made if we split and stored the result and checked its length.  `*shrug*`